src/components/Cluster/ClusterDetail/AppDetailsModal/InitialPane.js

```diff
@@ -82,7 +82,7 @@ const InitialPane = (props) => {
           <div className='labelvaluepair--label'>RELEASE STATUS</div>
           <div className='labelvaluepair--value code'>
             {props.app.status?.release?.status ? (
-              <Truncated as='span'>{props.app.status.release.status}</Truncated>
+              <span>{props.app.status.release.status}</span>
```

Comment:
This one will never need to be truncated, so removed the component.

But the app version can be very long in some cases (versions of test apps) so for those we do need some logic in the Truncated component to skip tooltips if they are not needed. (Added below)